test/unit/api/ws/workers/slaveToMasterSender.js

```diff
@@ -32,15 +28,16 @@ describe('SlaveToMasterSender', function () {
 				options: {
 					authKey: 'valid auth key'
 				}
-			}
+			},
+			sendToMaster: 'sendToMaster'
```

Comment:
Prefer putting the stub in here directly via `sinonSandbox.stub()`.

Reply:
This issue will be fixed as part of #1307